src/cmd-buildextend-installer

```diff
@@ -172,10 +168,10 @@ def generate_iso():
 
     run_verbose(genisoargs)
 
-    kernel_name = f'{base_name}-{args.build}-installer-kernel'
-    initramfs_name = f'{base_name}-{args.build}-installer-initramfs.img'
-    kernel_file = os.path.join(builddir, kernel_name)
-    initramfs_file = os.path.join(builddir, initramfs_name)
+    kernel_name = f"{bm['name']}-{args.build}-installer-kernel"
```

Comment:
For at least some of the common ones let's add a `@property` for e.g. `name` so that this can just be `bm.name` instead of the more noisy `bm['name']`?